### PR TITLE
Fix default contributor

### PR DESCRIPTION
**Pull request by dchiquito**
This should resolve https://github.com/dandi/dandi-cli/issues/682#issuecomment-868803784

### CODE REVIEW

File: dandiapi/api/views/dandiset.py [hunk 1]
```diff
@@ -110,15 +110,20 @@ def create(self, request):
...
-                'roleName': ['dandi:ContactPerson'],
-                'schemaKey': 'Person',
-                'includeInCitation': True,
-            }
-        ]
+        # Only inject a default contributor field if it is not specified in the metadata
+        metadata = {
+            'contributor': [
+                {
+                    'name': f'{request.user.first_name} {request.user.last_name}',
```
Comment (jwodder) [inline dandiapi/api/views/dandiset.py:117]: Dandischema requires personal names to be in the form "familyname, given names".

Comment (satra) [reply dandiapi/api/views/dandiset.py:117]: @dchiquito - i don't know if this comment from @jwodder got lost, but this will not pass validation.

Comment (dchiquito) [reply dandiapi/api/views/dandiset.py:117]: This test does not actually validate the metadata, but I will update this for consistencies sake in another PR.

Comment (dchiquito) [reply dandiapi/api/views/dandiset.py:117]: Thank you for pointing this out, I will fix it in another PR.

**Pull request merged**